aiatools/common.py

```diff
@@ -213,6 +213,16 @@ def children(self):
         return reduce(list.__add__, iter(self.values.values()), []) + \
                reduce(list.__add__, iter(self.statements.values()), [])
 
+    def has_mutation(self, name):
```

Comment:
Is `has_mutation` used anywhere?

Reply:
Never mind.  I just realized that this is the aiatools library and that the `has_mutation` method will be used by a client of the library.